DNN Platform/Website/Providers/DataProviders/SqlDataProvider/09.13.03.SqlDataProvider

```diff
@@ -7,3 +7,38 @@
 /*****       for {databaseOwner} and {objectQualifier}  *****/
 /*****                                                  *****/
 /************************************************************/
+
+/* update folder permissions view to include default value for portalID GHI:8240*/
+/*******************************************/
+
+if exists (select * from dbo.sysobjects where id = object_id(N'{databaseOwner}[{objectQualifier}vw_FolderPermissions]') and OBJECTPROPERTY(id, N'IsView') = 1)
+	DROP VIEW {databaseOwner}[{objectQualifier}vw_FolderPermissions]
+GO
+
+CREATE VIEW {databaseOwner}[{objectQualifier}vw_FolderPermissions]
+AS
+SELECT  FP.FolderPermissionID,
+        F.FolderID,
+        F.FolderPath,
+        ISNULL(F.PortalID, - 1) as PortalID,

```

Comment:
I have the same concern as others, this could potentially break other stuff, I would rather do the defaulting in the specific code

Reply:
I have modified the calling procedure instead as suggested in my previous comment